hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -892,16 +932,12 @@ static int xpadneo_input_configured(struct hid_device *hdev, struct hid_input *h
 	 * 0xB05 wireless Linux mode (Android mode)
 	 */
 	switch (xdata->idev->id.product) {
-#if 0
 	case 0x0B05:
-		/* FIXME What's the Windows ID of this controller? */
 		hid_info(hdev,
-			 "pretending XBE2 Windows wireless mode (changed PID from 0x%04X to 0x???)\n",
+			 "pretending XBE2 wireless Linux mode (changed PID from 0x%04X to 0x0B00)\n",
```

Comment:
This should read "pretending XBE2 wired mode" as this is what it does: `0x0B00` is the wired USB ID of the controller.